ql/src/java/org/apache/hadoop/hive/ql/exec/repl/DirCopyTask.java

```diff
@@ -182,6 +191,12 @@ public int execute() {
       Exception ex = new SecurityException(ErrorMsg.REPL_RETRY_EXHAUSTED.format(e.getMessage()), e);
       setException(ex);
       return ReplUtils.handleException(true, ex, work.getDumpDirectory(), work.getMetricCollector(), getName(), conf);
+    } finally {
+      String jobId = conf.get(ReplUtils.DISTCP_JOB_ID_CONF, ReplUtils.DISTCP_JOB_ID_CONF_DEFAULT);
```

Comment:
this will be printed in failure case

Reply:
The DirCopyTask Completed itself, The result of DistCp will be part of this param :
 ``+ "Result: {}",``

Reply:
Changed to status instead completed as suggested.